src/TypesAndInterfaces/custom-types.tsx

```diff
@@ -13,25 +16,72 @@ export const CIRCLE_SEARCH_ROUTE_NAME = "CircleSearch";
 export const CIRCLE_LIST_ROUTE_NAME = "CircleList";
 export const CIRCLE_DISPLAY_ROUTE_NAME = "CircleDisplay";
 export const CIRCLE_NAVIGATOR_ROUTE_NAME = "CircleRoutes";
+export const PRAYER_REQUEST_NAVIGATOR_ROUTE_NAME = "PrayerRequestRoutes";
 export const BOTTOM_TAB_NAVIGATOR_ROUTE_NAME = "BottomTabNavigator";
+export const PRAYER_REQUEST_LIST_ROUTE_NAME = "PrayerRequestList";
+export const PRAYER_REQUEST_DISPLAY_ROUTE_NAME = "PrayerRequestDisplay"
+export const CONTENT_NAVIGATOR_ROUTE_NAME = "ContentRoutes";
+export const PROFILE_SETTINGS_NAVIGATOR_ROUTE_NAME = "ProfileSettingsRoutes"
+
+export const BOTTON_TAB_NAVIGATOR_ROUTE_NAMES = [CIRCLE_NAVIGATOR_ROUTE_NAME, PRAYER_REQUEST_NAVIGATOR_ROUTE_NAME, CONTENT_NAVIGATOR_ROUTE_NAME, PROFILE_SETTINGS_NAVIGATOR_ROUTE_NAME];
+
+export enum PrayerRequestListViewMode {
+    RECIPIENT = "RECIPIENT",
+    OWNER = "OWNER"
+}
+
+export enum PrayerRequestRecipientViewMode {
+    CIRCLE = "CIRCLE",
+    USER = "USER"
+}
 
-export interface ProfileImageSettingsParams extends ParamListBase {
+export interface CallbackParam {
     callback:(() => void)
 }
 
+export interface PrayerRequestEditCallbackParam {
+    callback:((prayerRequestData:PrayerRequestResponseBody, prayerRequestListData:PrayerRequestListItem) => void)
+}
+
+export interface SelectListItem {
+    key: string | number, 
+    value: string | JSX.Element
+}
+
+export type FormSubmit = {
+    onHandleSubmit: () => void;
+}
+
+export type FormDataType = {
+    value: string | string[]
+}
+
+export type FormInputProps = {
+    fields:InputField[], 
+    defaultValues?:ProfileResponse | PrayerRequestResponseBody | number[], 
```

Comment:
Let's make it: `defaultValues?:any,`  since we're relying on config list anyways.

Reply:
done